db/system_keyspace.cc

```diff
@@ -1503,6 +1503,27 @@ future<> system_keyspace::cache_truncation_record() {
     });
 }
 
+    future<> system_keyspace::drop_truncation_rp_records() {
```

Comment:
Please remove cherry-pick marks from commits not in master. They'll dangle for others.

Reply:
ok